orc8r/cloud/go/services/streamer/const.go

```diff
@@ -16,5 +16,6 @@ package streamer
 import "syscall"
 
 const (
-	EAGAIN = syscall.EAGAIN
+	EAGAIN      = syscall.EAGAIN
+	ServiceName = "streamer"
```

Comment:
Why is the casing different? it looks like it used to be STREAMER?

Reply:
I solve this issue as well: https://github.com/magma/magma/issues/9829

Reply:
Here is my PR for lowercasing: https://github.com/magma/magma/pull/10377

Reply:
ohhhh I see! Looks good then :)